libraries/lib-numeric-formats/NumericConverter.h

```diff
@@ -63,7 +63,6 @@ class NUMERIC_FORMATS_API NumericConverter /* not final */
 
    enum Type {
       TIME,
-      ATIME, // for Audio time control.
```

Comment:
I see the enum, I think it's closed-endedness, and I would rather see registration remove the enum completely.

Maybe I'm about to see that.  No spoilers, please!

Reply:
This on stays. It can be quite easily removed though.